pkg/vz/vm_darwin.go

```diff
@@ -345,6 +345,19 @@ func attachFolderMounts(driver *driver.BaseDriver, vmConfig *vz.VirtualMachineCo
 		config.SetDirectoryShare(share)
 		mounts[i] = config
 	}
+
+	if driver.Yaml.Rosetta.Enabled {
+		logrus.Info("Setting up Rosetta share")
+		directorySharingDeviceConfig, err := createRosettaDirectoryShareConfiguration()
+		if errors.Is(err, errRosettaUnsupported) {
+			logrus.Warnf("Unable to configure Rosetta: %s", err)
+		} else if err != nil {
+			return err
```

Comment:
Probably, any error from `createRosettaDirectoryShareConfiguration` can be a warning.
e.g., failure of installation of Rosetta

Reply:
I'm still unsure of this. Is there other example cases where Lima fails to do setup and continues on anyways? It seems like continuing on would lead to unexpected results for users who may not notice the warnings, etc.

Reply:
> Is there other example cases where Lima fails to do setup and continues on anyways? 

Yes, in a bunch of places (`git grep -F .Warn`)

e.g.,

https://github.com/lima-vm/lima/blob/32baa6216f89ce4aaacbc254d13f31535bffeb2d/pkg/start/start.go#L213

Reply:
Gotcha. I've updated it to warn instead.